docs/src/main/asciidoc/README.adoc

```diff
@@ -1,18 +1,106 @@
+= Spring Framework on Google Cloud Platform
 
-Spring Cloud for Google Cloud Platform is a part of the Spring Cloud project and it makes integration with Google Cloud Platform (GCP) easy.
-It offers a convenient way to interact with GCP services using well-known Spring idioms and APIs.
-
-== Features
-
-* PubSub
-* Storage
-* SQL
-* Trace
-* Logging
-* Cloud Config
-* Cloud Spanner
-* Cloud Datastore
-* Cloud Foundry
-* MemoryStore
-* Security IAP
-* Vision
+This project makes it easy for Spring users to run their applications on Google Cloud Platform.
+You can check our project website http://cloud.spring.io/spring-cloud-gcp[here].
+
+For a deep dive into the project, refer to the
+https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.RELEASE/reference/htmlsingle/[Spring
+Cloud GCP 1.0 Reference Document].
```

Comment:
What's wrong with one sentence per line?

Reply:
I didn't modify this part. I just copy-pasted from what it was before.

Reply:
Good, but what's wrong with our code style rules and why just don't fix a violation since you are already here?
I'm OK do not fix anything we don't touch, but this is a change and it doesn't follow our rules.
Anyone then can say: "This is not my code why should I fix it?"
Then we really won't be able to reach a single solid code style since no one is going to pursue it.

Not sure why my point about pursuing a simplicity for the contribution because we have a single source of code style is so hard to follow...

Thanks for understanding